tools/cldr-apps/src/main/webapp/css/redesign.css

```diff
@@ -243,17 +243,39 @@ h3.collapse-review > span:first-child {position: relative; top: -220px;display:
 	border: 1px solid #0FF;
 }
 .data .visible-mark::before{
-	content: '<';
+	content: '❰';
```

Comment:
The downside of having this not in the text is that people can't search on page for it.

Reply:
same with other one.

Reply:
it's to make the display more flexible.

Reply:
Right. But people can't search for it. And the slight advantage of our being able to update with a CSS change instead of a code change isn't worth sacrificing that.

However, in the code they can be constants, so it is easy to change in one place.